src/components/products/Product.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Stack } from "@chakra-ui/react";
+import Ingredient from "../recipes/Ingredient.tsx";
+import { useGetIngredientPrice } from "../hooks/useGetIngredientPrice.tsx";
+import PlusMinus from "../PlusMinus.tsx";
+import { FC } from "react";
+import { NewIngredient } from "../types.ts";
+import { useMyStore } from "../store/store.tsx";
+
+type Props = {
+  ingredient: NewIngredient;
+};
+
+const Product: FC<Props> = ({ ingredient }) => {
```

Comment:
Jak si psala, taky se mi to moc nezamlouva (ale ted nevim jak z toho vem). Pojdme se nad tim zkuset zamyslet spolu.